.github/workflows/prod.yml

```
@@ -65,39 +131,21 @@ jobs:
       run: |
         docker push $AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com/$REPO_NAME:latest
 
-    - name: Notify Slack
+    - name: Send Slack notification on success
+      if: success()
       run: |
```

actionlint (1.7.8)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)